pkgs/development/libraries/arrow-cpp/default.nix

```
@@ -107,15 +117,24 @@ stdenv.mkDerivation rec {
     ninja
     autoconf # for vendored jemalloc
     flatbuffers
-  ] ++ lib.optional stdenv.isDarwin fixDarwinDylibNames;
+  ] ++ lib.optional enableCuda [
```

```suggestion
  ] ++ lib.optionals enableCuda [
```

See https://discourse.nixos.org/t/lib-optional-considered-harmful/7081

---

Done.